auth.go

```diff
@@ -6,10 +6,11 @@ import (
 
 	pb "github.com/13k/go-steam-resources/protobuf/steam"
 	"github.com/13k/go-steam-resources/steamlang"
+	"google.golang.org/protobuf/proto"
```

Comment:
Sorry about this change, Goland sort the imports by external then internal packages...